### PR TITLE
Bug 1594064 - Show framework in graph legend

### PR DESCRIPTION
**Pull request by airimovici**

### CODE REVIEW

**Comment by airimovici** [comment]
badge-on-legend

File: ui/perfherder/graphs/LegendCard.jsx [hunk 1]
```diff
@@ -86,6 +87,10 @@ const LegendCard = ({
     resetParams(newData);
   };
 
+  const getFrameworkName = framwork_id => {
+    const framework = frameworks.find(item => item.id === framwork_id);
+    return framework ? framework.name : 'no-framework';
```
Comment (ionutgoldan) [inline ui/perfherder/graphs/LegendCard.jsx]: Could we rename `'no-framework'` to `'unknown framework'`?

Comment (ionutgoldan) [reply ui/perfherder/graphs/LegendCard.jsx]: Otherwise, this looks good.

File: ui/perfherder/graphs/LegendCard.jsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ const LegendCard = ({
   updateState,
   updateStateParams,
   selectedDataPoint,
+  frameworks,
   colors,
 }) => {
```
Comment (ionutgoldan) [inline ui/perfherder/graphs/LegendCard.jsx]: Please provide some basic coverage, to make sure the framework name is displayed.

**Comment by ionutgoldan** [comment]
@airimovic the code looks good. Please fix the linting errors.

**Pull request merged**